front/app/components/Sharing/SharingButtons/index.tsx

```diff
@@ -317,9 +336,31 @@ const SharingButtons = memo(
         </Email>
       ) : null;
 
+    const copylink = (
+      <CopyLink
```

Comment:
Should the button be disabled ones the link is copied?

Reply:
I chatted with Ben about the response shown to users, and the text changing from "Copy link" to "Link copied" seems to be aligned with other apps. I'm thinking it could be good to have it still active in case they copy something else and then want to re-copy without reloading the dropdown or page maybe?